Documentation/supported-mod-list.md

```diff
@@ -18,324 +21,328 @@
+61. [Edge of Darkness](https://www.moddb.com/mods/hl-edge-of-darkness)
+62. [Edge of Death - Series 1](https://twhl.info/vault.php?map=5088)
+63. [Engineering Mankind](https://twhl.info/competitions.php?results=13)
+64. [Episode Power Plant and China](https://www.artpeter.net/Data/HalfLife/Hl.php)
+65. [Episode Secret Weapon](https://www.artpeter.net/Data/HalfLife/Hl.php) (you will face a serious inner bug at the second map if you try to play this mod on hard difficulty)
+66. [Escape by Leo Ring](https://www.runthinkshootlive.com/posts/escape-by-leo-ring/)
+67. [Escape from the Egyptian Tomb( RacerX Compo 12 Egyptian Museum )](https://twhl.info/vault.php?map=2368)
+68. [Espionage](https://www.runthinkshootlive.com/posts/espionage/)
+69. [Exodus](https://www.moddb.com/games/half-life/addons/exodus-1)
+70. [Exodus 2](https://www.moddb.com/games/half-life/addons/exodus-2)
+71. [Failure](https://www.moddb.com/games/half-life/addons/failure)
+72. [Faraon](https://hl.loess.ru/?mod=345)
+73. [Fate Reversal v1.1](https://www.moddb.com/mods/fate-reversal)
+74. [Fathom 2.4](https://twhl.info/vault.php?map=4759)
+75. [Freeman](https://www.runthinkshootlive.com/posts/freeman/)
+76. [Freeman's Escape](https://www.moddb.com/mods/freeman-escape-french) (2 maps by *JiggZ*)
+77. [Freeman's Fight](https://www.runthinkshootlive.com/posts/freemans-fight/)
+78. [Freeman's Return](https://www.moddb.com/games/half-life/addons/freemans-return-v12)
+79. [Freeman's Return 2](https://www.moddb.com/games/half-life/addons/freemans-return-2-v11)
+80. [Freeman's Revenge](https://www.runthinkshootlive.com/posts/freemans-revenge/)
+81. [Freeman's Tomb( The Crypt )](https://twhl.info/vault.php?map=3787)
+82. [G-Invasion v2.5](https://www.moddb.com/mods/g-man-invasion) aka G-Man Invasion (there is an inner crash bug on final titles: some text lines are too long and cannot be properly displayed)
+83. [G-Man House](https://hl.loess.ru/?mod=404)
+84. [GameStar( Episode 4 GameStar )](https://www.runthinkshootlive.com/posts/gamestar/)
+85. [Gateway](https://www.moddb.com/mods/gateway)
+86. [Gateway 2](https://www.moddb.com/mods/gateway-2)
+87. [Gina's Adventures](https://www.moddb.com/mods/ginas-adventures)
+88. [Gman Island Part 1](https://www.runthinkshootlive.com/posts/gman-island-part-1/)
+89. [Gman Island Part 2](https://www.runthinkshootlive.com/posts/gman-island-part-2/)
+90. [Graber v1.1](https://www.moddb.com/mods/graber-half-life-mod)
+91. [Graf War](https://www.runthinkshootlive.com/posts/graf-war/)
+92. [Ground Zero](https://www.moddb.com/mods/half-life-ground-zero) by *Derek 'Hellfire' McBurney*
+93. [Gut Reaction](https://www.moddb.com/mods/gut-reaction)
+94. [Half-Life C.A.G.E.D.](https://store.steampowered.com/app/679990/HalfLife_Caged/) (set **-num edicts 4096** in command-line parameters for successful start)
```

Comment:
Wait, do we even have `-numedicts` command line arg? 🤔 

Anyway, there is a typo `-num edicts`, with a space.